robot_smach_states/src/robot_smach_states/navigation/constraint_functions/arms_reach_constraints.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+# System
+import math
+
+# ROS
+import rospy
+from geometry_msgs.msg import Point
+
+# TU/e Robotics
+from cb_planner_msgs_srvs.msg import OrientationConstraint, PositionConstraint
+
+
+def arms_reach_constraint(pose_designator, arm_designator, look=True):
+    """
+    Position so that the arm can reach the position/entity
+    :param pose_designator: designator that resolves to a FrameStamped
```

Comment:
An empty line between the description and the lines that start with : is required for the doc processor afaik.

Reply:
Done